src/librustc_resolve/lib.rs

```diff
@@ -3512,6 +3635,62 @@ fn path_names_to_string(path: &Path, depth: usize) -> String {
     names_to_string(&names[..])
 }
 
+/// When an entity with a given name is not available in scope, we search for
+/// entities with that name in all crates. This method allows outputting the
+/// results of this search in a programmer-friendly way
+fn show_candidates(session: &mut DiagnosticBuilder,
+                   span: &syntax::codemap::Span,
+                   candidates: &SuggestedCandidates) -> () {
```

Comment:
There's no need to write out the unit return type -- this line can just be `candidates: &SuggestedCandidates) {`